### DISCUSSION TITLE
Payment

### DISCUSSION THREAD

**Discussion by thongjang** [community/community#145840] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I got an error while paying "Your account is currently locked from purchases. Please update your payment information." Please help me.

**Comment by goldentalent626** [comment]
What kind of payment gateway are you using now?

**Reply by thongjang** [reply]
I use Credit or debit card method.

**Reply by thongjang** [reply]
![image](https://github.com/user-attachments/assets/ef14d3ea-204f-4692-9db5-acbffd4caaef)
